### PR TITLE
Add ui tests

### PR DESCRIPTION
**Pull request by damien-urruty-sonarsource**

### CODE REVIEW

File: .project [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (henryju) [inline .project]: I missed that earlier, but we usually don't commit IDE specific files in our repositories. See for example https://github.com/SonarSource/sonarlint-eclipse/blob/master/.gitignore

File: platform.target [hunk 2]
```diff
@@ -6,6 +6,8 @@
 			<repository location="http://download.eclipse.org/releases/2020-03"/>
 			<unit id="org.eclipse.platform.ide" version="0.0.0"/>
 			<unit id="org.eclipse.jdt.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.swtbot.eclipse.feature.group" version="0.0.0"/>
```
Comment (henryju) [inline platform.target:9]: Interesting, I was not aware SWTBot was part of the Eclipse release update site...

**Review by henryju — APPROVED** [review]
Minor comment then LGTM. Be sure build is passing on Azure before merging ;)

File: .project (hunk 1 above)
Comment (damien-urruty-sonarsource) [reply .project]: I thought they were important. Removed them

**Comment by damien-urruty-sonarsource** [comment]
@henryju I think you can have another look

**Pull request merged**